<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IDE_CFactorial</class>
 <widget class="QMainWindow" name="IDE_CFactorial">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IDE_C!</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="EditorC">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>551</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RunButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QTableView" name="RamLiveView">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>41</y>
      <width>491</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="StepButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Step</string>
    </property>
   </widget>
   <widget class="QLabel" name="RamLiveViewTittle">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>361</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Ram Live View</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="StdOut">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>1081</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="StdOutTittle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>StdOut</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ApplicationLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>1081</width>
      <height>131</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="ApplicationLogTittle">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Application log</string>
    </property>
   </widget>
   <widget class="QLabel" name="TituloEditor">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Editor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StopButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAppLog">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>500</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearStdOut">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>330</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
